types/index.d.ts

```diff
@@ -86,6 +85,27 @@ declare namespace Handlebars {
 
   export function noConflict(): typeof Handlebars;
 
+  export interface ExceptionNode {
```

Comment:
I don't think we need this interface. The second parameter should be a `AST.Node`